lib/solargraph/rbs_map/conversions.rb

```diff
@@ -38,25 +38,6 @@ def load_environment_to_pins(loader)
         environment = RBS::Environment.from_loader(loader).resolve_type_names
         cursor = pins.length
         environment.declarations.each { |decl| convert_decl_to_pin(decl, Solargraph::Pin::ROOT_PIN) }
-        added_pins = pins[cursor..-1]
-        add_back_implicit_pins(added_pins)
-      end
-
-      # @param added_pins [::Enumerable<Pin>]
-      # @return [void]
-      def add_back_implicit_pins(added_pins)
-        added_pins.each do |pin|
-          pin.source = :rbs
```

Comment:
@castwide: I think this removal was unintentional - it affected one of our typechecks by suppressing it - I've been diffing the baseline strict typecheck results over time, and some warnings disappeared this morning when I merged in the latest master.  

See https://github.com/castwide/solargraph/pull/849 for details and a fix